[2025-03-24T12:53:30.096+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:53:30.119+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data manual__2025-03-24T12:53:27.368160+00:00 [queued]>
[2025-03-24T12:53:30.130+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data manual__2025-03-24T12:53:27.368160+00:00 [queued]>
[2025-03-24T12:53:30.131+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T12:53:30.147+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): download_source_data> on 2025-03-24 12:53:27.368160+00:00
[2025-03-24T12:53:30.154+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=84) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T12:53:30.157+0000] {standard_task_runner.py:72} INFO - Started process 85 to run task
[2025-03-24T12:53:30.157+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'download_source_data', 'manual__2025-03-24T12:53:27.368160+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmp0hblokjq']
[2025-03-24T12:53:30.159+0000] {standard_task_runner.py:105} INFO - Job 150: Subtask download_source_data
[2025-03-24T12:53:30.216+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.download_source_data manual__2025-03-24T12:53:27.368160+00:00 [running]> on host 244eeec36139
[2025-03-24T12:53:30.339+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='download_source_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-24T12:53:27.368160+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-24T12:53:27.368160+00:00'
[2025-03-24T12:53:30.340+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:53:30.387+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T12:53:30.388+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 2025-03']
[2025-03-24T12:53:30.401+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T12:53:30.404+0000] {subprocess.py:106} INFO - 2025-03
[2025-03-24T12:53:30.404+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T12:53:30.435+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:53:30.436+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=download_source_data, run_id=manual__2025-03-24T12:53:27.368160+00:00, execution_date=20250324T125327, start_date=20250324T125330, end_date=20250324T125330
[2025-03-24T12:53:30.496+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:53:30.532+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-24T12:53:30.534+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
